Profiles - ZIB 2017/zib-GeneralMeasurement.xml

```diff
@@ -61,7 +61,18 @@
         <valueString value="default: final" />
       </extension>
       <path value="Observation.status" />
-      <comment value="A ConceptMap (https://simplifier.net/resolve?canonical=http://nictiz.nl/fhir/ConceptMap/ResultStatusCodelist-to-ObservationStatus) is available that maps ResultStatusCodelist to the ObservationStatus valueset." />
+      <comment value="A ConceptMap (http://nictiz.nl/fhir/ConceptMap/ResultStatusCodelist-to-ObservationStatus) is available that maps ResultStatusCodelist to the ObservationStatus valueset." />
```

Comment:
Didn't we decide to put this in the mapping.comment?

Reply:
There is no mapping present on element Observation.status. Therefore, the location didn't change